podman/containers/__init__.py

```diff
@@ -0,0 +1,315 @@
+"""images provides the operations against images for a Podman service."""
+import json
+import logging
+
+import podman.errors as errors
+
+
+def make_call(api, endpoint, method='GET', params=None, body=None):
```

Comment:
Maybe this should go on the api class instead.

Reply:
That is logical.  Moved to api class and made the appropriate reference updates.